doc/source/index.rst

```diff
@@ -25,21 +25,31 @@ Sections
    auto_examples/index
    release_notes_and_installation
    core_developer
+   skips/index
+   values
 
 .. list-table::
    :class: contentstable
 
    * - `Overview <overview.html>`_
 
-       Introduction to skimage.
+       Introduction to scikit-image.
```

Comment:
Are these "short summaries" necessary? The names of the sections seem to speak for themselves.

Reply:
The short summaries are there already, not added by this PR. Let's leave their update/removal to later work, as this PR is already contentious enough.